src/views.coffee

```diff
@@ -112,6 +112,17 @@ FormRenderer.Views.NonInputResponseField = FormRenderer.Views.ResponseField.exte
     @$el.html JST['partials/non_input_response_field'](@)
     @
 
+FormRenderer.Views.ResponseFieldPrice = FormRenderer.Views.ResponseField.extend
+  field_type: 'price'
+  events:
+    'blur [data-js-price-cents]': 'formatCents'
+
+  formatCents: ->
+    @$input = @$el.find('input[data-js-price-cents]')
```

Comment:
If you pass the `event` to this function, you should be able to get the DOM node via `event.target`.